test/unit/spec/MfaVerify_spec.js

```diff
@@ -1446,7 +1479,11 @@ function (Okta,
           expect(test.button.trimmedText()).toEqual('Call');
           expect(test.form.answerField().val()).toEqual('123456');
           test.form.makeCall().click();
-          return tick().then(function () {
+          return Expect.wait(function () {
+            return test.form.answerField().val() === '';
```

Comment:
nit: we could also wait for an ajax call. That way we check the field value after the call is made, and not in the wait condition. Similar comment in other tests too. the reason is that the test will fail in a `wait` condition instead of failing in an `expect`.

But this can be done later too, what you have is already much better than `tick`

Reply:
I'm almost certain I started with waiting for the ajax call. But I believe there was some extra async action after the ajax call which required continued waiting